### PR TITLE
Continuous deployment

### PR DESCRIPTION
**Pull request by pxeger**
I used the word "funky" in a few places just to be non-generic in my descriptions of the system. It doesn't mean anything in particular.

### CODE REVIEW

**Comment by pxeger** [comment]
Plan:

- merge `cd` -> `finishing-touches`
- merge `finishing-touches` -> `fresh-beginnings`
- rename `fresh-beginnings` -> `main` (or maybe call it `develop` or something like that)
- rename `master`  (old branch) -> `old`
- branch from `main` to `production`, and update that branch when we want to update the website

File: flask_app.py [hunk 1]
```diff
@@ -129,19 +134,12 @@ def oeis():
...
-        origin.pull()
-        return "Updated PythonAnywhere successfully", 200
+    key = request.args.get("key", "")
+    if compare_digest(sha256(key.encode()).hexdigest(), FUNKY_PASSWORD_HASH):
+        import os
+        os.system("/home/Vyxal/mysite/funky_upgrade.sh")
+        return "ok", 200
     else:
-        return "Wrong event type", 400
+        return "no", 403
```
Comment (cgccuser) [inline flask_app.py:145]: Could we perhaps write something more descriptive than "no"?

**Review by cgccuser — APPROVED** [review]
Sounds like a good plan to me, perhaps we could call the development branch `dev` because it's golfier.

**Review by Lyxal — APPROVED** [review]
Looks good to me!

**Pull request merged**